errors.toml

```diff
@@ -0,0 +1,405 @@
+error = "canal decode failed"
+
+["CDC:ErrCanalEncodeFailed"]
+error = "canal encode failed"
+
+["CDC:ErrCaptureCampaignOwner"]
+error = "campaign owner failed"
+
+["CDC:ErrCaptureNotExist"]
+error = "capture not exists, key: %s"
+
+["CDC:ErrCaptureRegister"]
+error = "capture register to etcd failed"
+
+["CDC:ErrCaptureResignOwner"]
+error = "resign owner failed"
+
+["CDC:ErrCaptureSuicide"]
+error = "capture suicide"
+
+["CDC:ErrChangeFeedAlreadyExists"]
+error = "changefeed already exists, key: %s"
+
+["CDC:ErrChangeFeedNotExists"]
+error = "changefeed not exists, key: %s"
+
+["CDC:ErrChangefeedAbnormalState"]
+error = "changefeed in abnormal state: %s, replication status: %+v"
+
+["CDC:ErrCheckClusterVersionFromPD"]
+error = "failed to request PD"
+
+["CDC:ErrCheckDirWritable"]
+error = "check dir writable failed"
+
+["CDC:ErrCodecDecode"]
+error = "codec decode error"
+
+["CDC:ErrCreateMarkTableFailed"]
+error = "create mark table failed"
+
+["CDC:ErrDDLEventIgnored"]
+error = "ddl event is ignored"
+
+["CDC:ErrDatumUnflatten"]
+error = "unflatten datume data"
+
+["CDC:ErrDecodeFailed"]
+error = "decode failed: %s"
+
+["CDC:ErrDecodeRowToDatum"]
+error = "decode row data to datum failed"
+
+["CDC:ErrEncodeFailed"]
+error = "encode failed: %s"
+
+["CDC:ErrEventFeedAborted"]
+error = "single event feed aborted"
+
+["CDC:ErrEventFeedEventError"]
+error = "eventfeed returns event error"
+
+["CDC:ErrExecDDLFailed"]
+error = "exec DDL failed"
+
+["CDC:ErrFetchHandleValue"]
+error = "can't find handle column, please check if the pk is handle"
+
+["CDC:ErrFileSinkCreateDir"]
+error = "file sink create dir"
+
+["CDC:ErrFileSinkFileOp"]
+error = "file sink file operation"
+
+["CDC:ErrFileSinkMetaAlreadyExists"]
+error = "file sink meta file already exists"
+
+["CDC:ErrFileSorterDecode"]
+error = "decode failed"
+
+["CDC:ErrFileSorterEncode"]
+error = "encode failed"
+
+["CDC:ErrFileSorterInvalidData"]
+error = "invalid data"
+
+["CDC:ErrFileSorterOpenFile"]
+error = "open file failed"
+
+["CDC:ErrFileSorterReadFile"]
+error = "read file failed"
+
+["CDC:ErrFileSorterWriteFile"]
+error = "write file failed"
+
+["CDC:ErrFilterRuleInvalid"]
+error = "filter rule is invalid"
+
+["CDC:ErrGRPCDialFailed"]
+error = "grpc dial failed"
+
+["CDC:ErrGetAllStoresFailed"]
+error = "get stores from pd failed"
+
+["CDC:ErrGetRegionFailed"]
+error = "get region failed"
+
+["CDC:ErrGetStoreSnapshot"]
+error = "get snapshot failed"
+
+["CDC:ErrGetTiKVRPCContext"]
+error = "get tikv grpc context failed"
+
+["CDC:ErrIndexKeyTableNotFound"]
+error = "table not found with index ID %d in index kv"
+
+["CDC:ErrInternalServerError"]
+error = "internal server error"
+
+["CDC:ErrIntersectNoOverlap"]
+error = "span doesn't overlap: %+v vs %+v"
+
+["CDC:ErrInvalidAdminJobType"]
+error = "invalid admin job type: %d"
+
+["CDC:ErrInvalidChangefeedID"]
+error = "bad changefeed id, please match the pattern "^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$", eg, "simple-changefeed-task""
```

Comment:
Seems the error text needs to be wraped with `'''`.

Reply:
Is it better the generator handles the escape, such as translating `"` to `\"`

Reply:
Fixed in https://github.com/pingcap/tiup/pull/851/commits/ad64b15d57de7bd4a6056115aa64484f6eae8847